src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -84,10 +84,6 @@ public static void main(String[] args) throws Exception {
         if (staticLibs != null) {
```

Comment:
In such cases Optional is cleaner I think:
`Optional.of(staticLibs).ifPresent(config::setStaticLibs)`

Reply:
ok